src/django/api/constants.py

```
@@ -203,7 +203,7 @@ class APIErrorMessages:
 
 
 class FacilitiesDownloadSettings:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)